.changeset/chore-deploy-old-workflow.md

```diff
@@ -0,0 +1,6 @@
+---
+'react-magma-docs': patch
+---
+
+chore: added disabled workflow publish-old
+chore: sync main with dev after PR "build: Release landing page updates #1749"
```

Comment:
the sync main to dev also needs to include updating the landing package version (example https://github.com/cengage/react-magma/pull/1719/files) and removing the changeset